### DISCUSSION TITLE
xterm@5.2.1:  There is a bug in the input character

### DISCUSSION THREAD

**Discussion by BensonLam** [sourcelair/xterm.js#4708] [Q&A]
## Details
- Browser and browser version: all browser
- xterm.js version: xterm@5.2.1

### Steps to reproduce

1. the textarea in the input character  cannot wrap

**Comment by BensonLam** [comment]
![image](https://github.com/xtermjs/xterm.js/assets/26159182/c697e764-ff4e-43c7-951e-ccc3f967ab51)

**Comment by BensonLam** [comment]
pls tell me how to fix

**Comment by Tyriar** [comment]
See https://github.com/xtermjs/xterm.js/wiki/FAQ#characters-are-being-overwritten-when-wrapped-to-a-new-line
